website/docs/events/m3-slides/.slides/next-steps.md

```diff
@@ -2,3 +2,12 @@
 
 ## Next Steps
 
+----
+
+### Next Steps
+
+- Solve all issues
+- Develop new assignments
+- Reiterate content
+- Smiley Coin Integration
```

Comment:
I would develop on this a bit. Mention Mihai's recent progress.

Reply:
Yes, with words.